testsuite/gnat2goto/tests/cfg_test_11/cfg_test_11.adb

```diff
@@ -0,0 +1,41 @@
+procedure CFG_Test_11 (X : in Integer; Y : in out Integer) is
+   I : Integer;
+   J : Integer;
+   K : Integer;
+begin
+   Y := 0;
+   
+   I := 0;
+Outer : 
+    while I <= X loop
+       
+       J := 0;
+   Middle :
+       while J <= X loop
+	  
+	  K := 0;
+      Inner:
+	  while K <= X loop
+	     
+	     exit Inner when I - J - K = 17;
+	     
+	     Y := Y + I - J + K;
+	     
+	     exit Middle when I + J - K = 23;
+	     
+	     K := K + 1;
+	     
+	     exit Outer when I + J + K = 42;
+	  end loop Inner;
```

Comment:
Is the indentation correct here?

Reply:
I have used `gnatpp` to pretty print the code.  It looks better to me now.